coredns/README.md

```diff
@@ -0,0 +1,68 @@
+# CoreDNS Integration
+
+## Overview
+Get metrics from CoreDNS in real time to:
+
+* Visualize and monitor DNS failures and cache hits/misses
+## Setup
```

Comment:
Since this integration is in integration-core. 
The CoreDNS check is packaged with the Agent? if so, no need to manually download `check.py` and `check.conf` files.